packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -35,7 +35,13 @@ export const userOwnsAndInGroup = (group: string) => {
   }
 }
 
-export const userIsSharedOn = (currentUser: DbUser|UsersMinimumInfo|null, document: PostsList|DbPost): boolean => {
+export interface SharableDocument {
+  coauthorStatuses: DbPost["coauthorStatuses"]
+  shareWithUsers: DbPost["shareWithUsers"]
+  sharingSettings: DbPost["sharingSettings"]
+}
```

Comment:
This interface has all these fields being required but the `isSharable` check only checks for at least one of them, particularly I'm pretty sure coauthorStatuses can be null while the other two aren't. Apparently you can make a type that requires at least one field to be set: https://stackoverflow.com/questions/40510611/typescript-interface-require-one-of-two-properties-to-exist

I would also be fine with just removing this type as it's currently just giving a bit of a false sense of security